app/shared/models/git_repo.rb

```diff
@@ -61,6 +63,16 @@ class GitRepo
     attr_accessor :callback
 
     class << self
+      def pushes_disabled?
```

Comment:
That seems like common ENV variable flag handling - don't we already have other boolean flag ENV variables where we can re-use the code? Maybe we should move this as a helper method to `FastlaneCI::EnvironmentVariables`?

Reply:
I didn't want to put it there because it's not a general property, it's very specific to this one class, so it didn't make sense to me to put them there. This is more of a fastlane.ci developer-specific switch. I wouldn't want our normal ci users to think they should muck around with it.

Reply:
I just updated the class comment for `EnvironmentVariables` to note this.